$NetBSD: patch-aa,v 1.14 2007/01/29 23:05:51 dmcmahill Exp $

name space collision with a #define in /usr/include/term.h on solaris-2.9

--- frontend/brhist.c.orig	2006-09-24 09:53:31.000000000 -0400
+++ frontend/brhist.c
@@ -429,7 +429,7 @@ stats_line(double *stat)
 void
 brhist_disp(const lame_global_flags * gf)
 {
-    int     i, lines = 0;
+    int     i, mylines = 0;
     int     br_hist[BRHIST_WIDTH]; /* how often a frame size was used */
     int     br_sm_hist[BRHIST_WIDTH][4]; /* how often a special frame size/stereo mode commbination was used */
     int     st_mode[4];
@@ -456,13 +456,13 @@ brhist_disp(const lame_global_flags * gf
         if (most_often < br_hist[i])
             most_often = br_hist[i];
         if (br_hist[i])
-            ++lines;
+            ++mylines;
     }
 
     for (i = 0; i < BRHIST_WIDTH; i++) {
         int     show = br_hist[i];
 #ifdef RH_HIST
-        show = show && (lines > 1);
+        show = show && (mylines > 1);
 #endif
         if (show
             || (i >= brhist.vbr_bitrate_min_index
